src/components/category-vocabs/CategoryDetail.js

```diff
@@ -1,29 +1,28 @@
 import React from "react";
 import { Card, CardBody, CardTitle } from "shards-react";
-import { Link } from "react-router-dom";
 import { useTranslation } from "react-i18next";
 
 const CategoryDetail = ({ categoryItem }) => {
   const { t } = useTranslation();
   return (
-    <Link to={`/category/${t(categoryItem.title)}`} style={{ textDecoration: 'none', color: 'inherit'}}>
```

Comment:
What could be the reason not to use react route link? 
If we use react router link, we can navigate to the page without refresh the whole page. Would it still behave the same if we use normal link?

Reply:
i realized when react route Link is used, the back button doesn't seem to work. But I'll look up on the problem now and figure out the reason for it. Tq!

Reply:
I've fixed the react route Link problem by adding 'exact' in Route and also hid the BackButton from home page